packages/create-turbo/src/commands/create/index.ts

```diff
@@ -20,6 +20,7 @@ import { TransformError } from "../../transforms/errors";
 import { isDefaultExample } from "../../utils/isDefaultExample";
 import * as prompts from "./prompts";
 import type { CreateCommandArgument, CreateCommandOptions } from "./types";
+import { updateCommandsInREADME } from "./updateCommandsInREADME";
```

Comment:
Let's add this as a transformer (inside `transforms`) instead of a one-off util

Reply:
I've added this script at packages/create-turbo/src/transforms/update-commands-in-readme.ts
Is there anything else I should take care of as per naming conventions?